frontend/cypress/integration/user_frontend_test.js

```diff
@@ -0,0 +1,82 @@
+/// <reference types="cypress" />
+
+context('Visit conquery as users with different permissions', () => {
```

Comment:
We're using prettier to auto-format js / ts code on save. Could you configure your editor to use prettier? (and use our installed / configured prettier – see `package.json`)

Reply:
I added that to the ReadMe aswell